### DISCUSSION TITLE
Any benefit to enabling predbat charge/discharge on fixed tariff

### DISCUSSION THREAD

**Discussion by mudge** [springfall2008/batpred#1198] [Q&A] [answered]
I’m a new predbat user having recently had 7.2 kWp PV system installed with a 9.6 kWh battery and iBoost (with the cut-in threshold set to 200W).

I’m still waiting for smart meters to be installed and, as such, am still on a fixed rate “loyal” Octopus tariff (though am planning to move to Octopus Flux as soon as I can). My GivEnergy Hybrid Inverter 5.0 Gen 3 is in its default “Eco” mode.

The [Configuration guide](https://springfall2008.github.io/batpred/configuration-guide/) recommends only using predbat’s “Monitor” mode in this setup but would there be any benefit to me switching it to manage charging given the solar forecast and load data or should I just sit tight and be patient until I’m on an export tariff before letting predbat manage the battery charge?

**Comment by gcoan** [comment, marked as answer]
Welcome to predbat!

If you are on a single flat rate day and night for your import electricity and are not getting paid for your export, then no, there's no real benefit to predbat managing your battery and the GivEnergy Eco mode will do everything you want - charge when there is solar and release the battery charge to the home when there is home demand that the solar can't meet (either in the day or overnight).

Having said that, it shouldn't do any harm if you turn on Charge or Charge and Discharge modes, and it will give you the opportunity to learn how predbat works, get your dashboards setup, learn how to override predbat if you want to, understand the plan, etc.   You'll get a plan if in Monitor mode anyway but a live plan where predbat is managing your battery should be more beneficial to play with.

Do you have gas heating for your hot water or is it electric only?  When you get your smart meter and then your export MPAN (which unfortunately can take up to 6 weeks after you apply for Flux), you may well find that its not worth using your iboost and instead its cheaper to heat the cylinder by gas.  You'll be getting ~16p/kWh from electricity export which could well be more than it costs to use the gas boiler.

**Reply by mudge** [reply]
Thanks for the prompt response, @gcoan. That’s all good to know. Given how long it’ll take before I’ll be able to export, is there any benefit to leaving predbat in Monitor mode until I have 7 days of load history before enabling it to see the “real” plans?

I have gas heating for my hot water too with the iBoost hooked up to the electric immersion heater (which was previously an unused “booster” switch). Interesting to hear it might not be worth it financially…

**Reply by gcoan** [reply]
You need at least a couple of days of history before predbat can meaningfully make predictions, but you don't need to leave it the full 7 days in Monitor mode.  Maybe leave it 3 days should be enough.

TBH I doubt that predbat will instruct your inverter to do all that much given your rates, but be interesting to see what it does do.
